frontend/src/modules/member/components/list/member-list-table.vue

```diff
@@ -336,6 +336,45 @@
                           >{{ email }}</a>
                         </div>
                       </el-tooltip>
+                      <el-popover
+                        v-if="scope.row.emails?.length > 3"
+                        placement="top"
+                        :width="400"
+                        trigger="hover"
+                        popper-class="support-popover"
+                      >
+                        <template #reference>
+                          <span
+                            class="badge--interactive hover:text-gray-900"
+                          >+{{ scope.row.emails.length - 3 }}</span>
+                        </template>
+                        <div class="flex flex-wrap gap-3 my-1">
+                          <el-tooltip
+                            v-for="email of scope.row.emails"
```

Comment:
I think Nuno's image might have induced an error, but the goal inside the popover is that it only displays the emails that are not visible on this list.

So let's say you have 5 emails. On the list it should display 3 and on the popover it should show the remaining 2 (that's why there's the +2 tag to indicate that 😄 )

![Screenshot 2023-09-13 at 16 19 54](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/8e107275-62b4-4edb-ba73-01263b1a7ad1)
![Screenshot 2023-09-13 at 16 19 57](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/8b9491ea-3f68-46d7-9c6d-d826c7a9cb48)

Reply:
got that 👍🏽 , will make the changes

Reply:
fixed